tests/input_files/ref/sohex8_multiscale_macro/xxx-vtk-files/structure-00000-0.vtu

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?> 
+<!-- 
+# vtk DataFile Version 3.0
+-->
+<VTKFile type="UnstructuredGrid" version="0.1" compressor="vtkZLibDataCompressor" byte_order="LittleEndian">
+  <UnstructuredGrid>
+    <FieldData>
+      <DataArray type="Float64" Name="TIME" NumberOfTuples="1" format="ascii">
+0.000000000000000e+00 
+      </DataArray>
+    </FieldData>
+
+    <Piece NumberOfPoints="24" NumberOfCells="3" >
+      <Points>
+        <DataArray type="Float64" NumberOfComponents="3" format="binary">
+AQAAAEACAABAAgAAcgAAAA==eNpjYACCGQv38INohgSb/yDQv2gPA1QcStuAaYYP+xlQAIz/YT9Yn8U2uDomEDVh8R40+T3o4rjMI46GAoQ7YebaM+DwF353YrjDHoc77YnzF0H/2cP0w/g43IPqDlzxgiseMfxDyJ8E4xnuXwATopJ3
```

Comment:
Please check in vtk data in `ascii` format (also applies to other files in this PR).